config/api.yml

```diff
@@ -1932,6 +1932,8 @@
         :identifier: service_admin
       - :name: delete
         :identifier: service_delete
+      - :name: add_resource
+        :identifier: service_edit
```

Comment:
@jntullo let's add this action on the collection too for completeness so we support bulk add_resources too.  Thanks.